planning/mission_planner/include/mission_planner/mission_planner_interface.hpp

```diff
@@ -18,10 +18,24 @@
 #include <rclcpp/qos.hpp>
 
 #include <autoware_planning_msgs/msg/pose_with_uuid_stamped.hpp>
+#include <autoware_planning_msgs/srv/set_pose_with_uuid_stamped.hpp>
+#include <std_srvs/srv/trigger.hpp>
 
 namespace mission_planner
 {
 
+struct SetMrmGoal
```

Comment:
how about here? SetMrmRoute?

Reply:
Thank you. Fixed it https://github.com/autowarefoundation/autoware.universe/pull/3299/commits/f02299e57941cc227df9013fa984cb59f9e3a504.